bbb-libreoffice/EULA.txt

```diff
@@ -0,0 +1,100 @@
+ END-USER LICENSE AGREEMENT FOR
```

Comment:
Maybe we should add a comment here making it clear that this EULA only applies to non-free variant?

Reply:
It is only displayed if the nonfree variant is selected.

Reply:
Ah, I think you mean it should still be in the file so that people looking at the source tree do not minterpret it to be the EULA for the image soruces?

Reply:
Yes, that was the idea, to avoid misinterpretations when looking source.